java/src/com/ibm/streamsx/topology/Topology.java

```diff
@@ -398,6 +398,48 @@ public Topology topology() {
         return graphItems;
     }
     
+    /**
+     * Includes a jar file, specified by the {@code location} String, into 
+     * the application runtime. For example, the following code includes the 
+     * myResource.jar jar file such that it can be used when running in a 
+     * DISTRIBUTED or STANDALONE context.
+     * <pre><code>
+     * Topology top = new Topology("myTopology");
+     * top.addThirdPartyDependency("./libs/myResource.jar");
+     * </pre></code> 
+     * For running embedded, simply adding the jar to the classpath when 
+     * compiling/running is sufficient.
+     * 
+     * @param location The location of a jar to be included in the application's
+     * runtime when submitting with a DISTRIBUTED or STANDALONE context.
+     */
+    public void addThirdPartyDependency(String location) {
+       JavaFunctional.addThirdPartyDependency(this, location); 
+    }
+    
+    /**
+     * Includes a class file into the application's runtime. The following 
```

Comment:
@wmarshall484 Isn't it doing more that including a class file? Namely it's including the complete set of classes that include the passed in class, which is the top-level 'classes; directory or the class's jar file.